Documentation/Signoffs/Safety.md

```diff
@@ -0,0 +1,93 @@
+
+# Safety Subsystem
+
+![ALT](https://github.com/Hawk652/Capstone-Guidance-Robot/blob/main/Documentation/Images/safety/safetyblockonly.png)
+
+#  Function of the Subsystem
+
+
+ The safety Subsystem will be tasked with obstacle avoidance, a manual override, and utilizing the LIDAR’s mapping capabilities. The sensors and button will be attached to the top of  the robot, that way it can utilize “seeing” capabilities. The sensors will gather information within the range (0.5 meters) of  robot and send this information to the Arduino or Raspberry PI.  To ensure that there will be no obstacle undetected, the LIDAR and Proximity sensor, ultrasonic  will be utilized to ensure redundancy. When an obstacle is detected within the range by the proximity sensor, ultrasonic sensor and the LIDAR, the information will be transmitted to the Main Control subsystem.  The LIDAR will be using its mapping capabilities to determine if the robot is too close to walls, doors, and corners. The proximity sensor will be using a 360 obstacle avoidance technique to determine the location of the obstacle. The DDS module will take the signal from a crystal oscillator and use it as a clock to create a buffer output waveform at a specific frequency that is set by the onboard microcontroller.  The Grove - Ultrasonic Distance Sensor will be using a range of  3cm - 350cm by using a non-contact form of measurement. 
+
+#  Constraints
+|     No. |Constraint  |  Origin  |
+|-|-|-|
+|1|  Shall have manual override (power will be cut from the motor)           |Supervisor:          Dr. Van Neste        |
+|2          |Shall maintain a distance of 0.5 m from obstacles and people           |Broader Implication         |
+
+#  Buildable Schematics
+##  Subsytsem Schematic
+
+![ALT](https://github.com/Hawk652/Capstone-Guidance-Robot/blob/main/Documentation/Images/safety/SafetySubsystemschematic.png)
+
+This schematic illustrates all the componts that will be interacting with one another in this system. The schematic shows how each of these will be connected.Each of these sections will be covered throughout this document. 
+
+##  Near Field Sensor (Proximity Sensor)
```

Comment:
it would be nice to show how you determined the size of the coils using the equations provided by van neste